### ISSUE TITLE
BUG: "cannot read property 'profile' of null

### ISSUE THREAD

**Issue by lpatmo** [codebuddiesdotorg/cb-v2-scratch#36]
Probably related to this commit: https://github.com/codebuddiesdotorg/cb-v2-scratch/commit/a0e3176b73a6329085044ae1d8984664dcd53c81

```
logged_in_using_codebuddies_team: function() {
    return Meteor.user().profile.team_id === "T04AQ6GEY";
  }
``` in `status/status-item.js` 

Error on the web console:
```
Exception in template helper: TypeError: Cannot read property 'profile' of null
    at Object.Template.statusItem.helpers.logged_in_using_codebuddies_team (http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:241:290)
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:30730
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:16487
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:31363
    at Function.e._withTemplateInstanceFunc (http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:94:4982)
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:31270
    at Object.o.call (http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:110:1878)
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:215:249
    at null.<anonymous> (http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:27429)
    at http://cbv2-staging2.meteor.com/373d5c59ccdf41421fe58eaf6471f0e441ca2334.js?meteor_js_resource=true:93:18465
```

**Comment by adanot** [comment]
I can take a look at this is nobody is already working on it. I have an idea of what might be going on.

**Comment by lpatmo** [comment]
@adanot Will assign it to you if you want to take a look. Thanks Adam!

**Comment by lpatmo** [comment]
@adanot Do you have thoughts on where you think the problem might be, without taking too much time to look? It's still stumping me.

**Comment by sgtquezada** [comment]
Is this issue fixed?

**Comment by lpatmo** [comment]
Not yet.

**Comment by sgtquezada** [comment]
Is the code committed? I don't see this in the `master` branch.  What is the expected behavior or outcome here?

**Comment by lpatmo** [comment]
@sgtquezada You should see the error log in the console if you sign out, then sign in back again via a Slack team that's not codebuddies. However, I could swear that I've seen it in console log even when I signed into the correct Slack...

**Comment by lpatmo** [comment]
I tried 

```
//fix for "cannot read property of 'null' error"
Meteor.publish(null, function() {
  return Meteor.users.find({_id: this.userId}, {profile.team_id: 1}});
});
```
but didn't get it to work. :/

**Comment by distalx** [comment]
Fixed #57
